ctf/check.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
Here is another way to put it: why is a batch script even needed?
Just open the terminal in the directory and run a lua script or something.
Afaik, even windows 7 support that.

There is no need for os dependent stuff.